java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -109,7 +110,7 @@ private OrtSession(long sessionHandle, OrtEnvironment environment, OrtAllocator
     super(sessionHandle);
     this.environment = environment;
     this.allocator = allocator;
-    this.defaultRunOptions = new RunOptions();
+    this.activeRunOptions = ConcurrentHashMap.newKeySet();
```

Comment:
Doesn't this set need a default mapped value to work (see https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ConcurrentHashMap.KeySetView.html#add-K-).

Reply:
It should work as it. I tested with:
```java
Set<String> set = ConcurrentHashMap.newKeySet();
System.out.println(set);
set.add("foo");
System.out.println(set);
set.add("bar");
System.out.println(set);
set.add("foo");
System.out.println(set);
set.add("baz");
System.out.println(set);
set.remove("foo");
System.out.println(set);
```
and got:
```
[]
[foo]
[bar, foo]
[bar, foo]
[bar, foo, baz]
[bar, baz]
```

Reply:
Ok cool. Someone should fix that doc, it still implies it needs a sentinel value even in the latest version.